mne/io/constants.py

```diff
@@ -852,3 +852,6 @@ def __setattr__(self, attr, val):  # noqa: D105
 
 # MNE annotations
 FIFF.FIFFB_MNE_ANNOTATIONS          = 3810  # annotations
+
+# MNE Metadata Dataframes
+FIFF.FIFFB_MNE_METADATA             = 3811  # metadata dataframes
```

Comment:
@jaeilepp can you look into adding this one along with the one from #4372:

https://github.com/mne-tools/mne-python/pull/4372/files#diff-d9d5ea0833eb382e8b37ec7af8280a16